docs/getting-started/configuration.md

```diff
@@ -0,0 +1,307 @@
+---
+id: index
+title: Getting Started, configuring Backstage
+description: Getting started with your initial Backstage configuration
+---
+
+This is part two of the Getting Started documentation of Backstage. The steps in
+this tutorial assume you've installed Backstage app from the npm repository,
+like in the [Getting Started guide](./index.md) and want to configure Backstage.
+
+At the end of this tutorial, you can expect:
+
+- Backstage to use a PostgreSQL database
+- You'll authenticate using one of the auth providers
+- The Backstage GitHub integration to be configured
+- You're able to use Software Templates
+
+### Prerequisites
+
+- Access to a Linux-based operating system, such as Linux, MacOS or
+  [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/)
+- An account with elevated rights to install prerequisites
```

Comment:
I got confused with "account" a bit as it could be GitHub account, Backstage account. But I think this means an `OS user`, doesn't it?